ush/ufsda/yamltools.py

```diff
@@ -307,7 +305,7 @@ def include_yaml(config):
 
 
 def replace_vars(config):
-    # use SOLO to replace variables in the configuration dictionary
+    # use wxflow to replace variables in the configuration dictionary
```

Comment:
I think wxflow Template needs imported here like it is in the other file

Reply:
I may be misunderstanding your comment but here's the change I committed to `yamltools.py`
```
@@ -3,7 +3,7 @@ import os
 import re
 import logging
 from ufsda.misc_utils import isTrue
-import wxflow
+from wxflow import YAMLFile, TemplateConstants, Template

 logging.basicConfig(format='%(asctime)s:%(levelname)s:%(message)s',
                     level=logging.INFO, datefmt='%Y-%m-%d %H:%M:%S')
```
Done at 1a689f8.